bots/image-refresh

```diff
@@ -86,15 +86,20 @@ TRIGGERS = {
         "fedora-30/selenium-chrome",
         "fedora-30/selenium-firefox",
     ],
+    "rhel-7-6": [
+        "rhel-7-6@cockpit-project/cockpit/rhel-7.6",
+    ],
     "rhel-7-7": [
         "rhel-7-7/firefox@weldr/cockpit-composer",
+        "rhel-7-7@cockpit-project/cockpit/rhel-7.7",
     ],
     "rhel-8-0": [
-        "rhel-8-0",
+        "rhel-8-0@cockpit-project/cockpit/rhel-8.0",
```

Comment:
Adding the new ones is great, but since these are very likely failing right now, let's enable these in a separate PR? I. e. I just fixed rhel-7.7 and  -8.1, but not rhel-8.0 yet. (And most probably we'll just drop it).